variety/VarietyWindow.py

```diff
@@ -1413,9 +1413,8 @@ def apply_clock(self, to_set):
     def apply_copyto_operation(self, to_set):
         if self.options.copyto_enabled:
             folder = self.get_actual_copyto_folder()
-            target_file = os.path.join(
-                folder, "variety-copied-wallpaper-%s.jpg" % Util.random_hash()
-            )
+            target_fname = "variety-copied-wallpaper-%s.%s" % (Util.random_hash(), os.path.splitext(to_set)[1])
```

Comment:
```suggestion
            target_fname = "variety-copied-wallpaper-%s%s" % (Util.random_hash(), os.path.splitext(to_set)[1])
```
`os.path.splitext()` preserves the `.` inside the extension, so this would generate paths like `variety-copied-wallpaper-0ddb071917324151b2406464fe836f1b..jpg`

Reply:
Got it - thanks :-)